Content.Server/Botany/SeedPrototype.cs

```diff
@@ -154,6 +154,28 @@ public class SeedData
 
     [DataField("potency")] public float Potency = 1f;
 
+    /// <summary>
+    ///     If true, cannot be harvested for seeds. Balances hybrids and
+    ///     mutations.
+    /// </summary>
+    [DataField("seedless")] public bool Seedless = false;
+
+    /// <summary>
+    ///     If true, rapidly decrease health while growing. Used to kill off
+    ///     plants with "bad" mutations.
+    /// </summary>
+    [DataField("viable")] public bool Viable = true;
+
+    /// <summary>
+    ///     If true, fruit slips players.
+    /// </summary>
+    [DataField("slip")] public bool Slip = false;
+
+    /// <summary>
+    ///     If true, fruits are sentient.
+    /// </summary>
+    [DataField("sentient")] public bool Sentient = false;
```

Comment:
not sure how i feel about hardcoding all of these possible variations into the seedprototype. eventually there's going to be a lot more of these, and they're going to want some variation (radius, color, degree of slipperiness...)

a solution I think we could do is something similar to `TraitPrototype` which just defines a component registry and adds everything onto the entity that spawns. so the seedprototype could just have a list of `BotanyTraitPrototype` IDs and it adds all the components from those on. that way you can have a lot more variation easily

Reply:
While this could be desirable in the future when there are more traits, hard coding this now separates the content changes from refactoring changes and reduces the diff of this PR.

While I'd be happy to work on a `TraitPrototype`, perhaps it could be better organized into a future PR.

Reply:
alright, i'm fine with that coming later